CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+- Fix: Cleanup lib/common/sem.cpp (#1056)
```

Comment:
```suggestion
```

Except in special cases, refactors like this one are not mentioned in the CHANGES_NEXT_RELEASE file, as they are internal and for Orion user (the one to which changelogs are addressed) is transparent.

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3869/commits/e4482579bd6b8125c68f3260d44d94f922e2cac6